test/filter/test_emoji.rb

```diff
@@ -1,7 +1,6 @@
 # ~*~ encoding: utf-8 ~*~
 path = File.join(File.dirname(__FILE__), "..", "helper")
 require File.expand_path(path)
-require 'ostruct'
```

Comment:
Does this belong in this PR? I don't really mind, just wondering about its significance. 😄